README.md

```diff
@@ -18,7 +18,7 @@
 [travis]: https://travis-ci.org/mcu-tools/mcuboot
 [license]: https://github.com/mcu-tools/mcuboot/blob/main/LICENSE
 
-This is MCUboot version 2.3.0-rc2
+This is MCUboot version 2.3.0
```

Comment:
need update in `boot/zephyr/VERSION` also

Reply:
@d3zd3z it is still missing